cmake/modules/FindQt5.cmake

```diff
@@ -0,0 +1,20 @@
+if (APPLE)
```

Comment:
I don't think we should reintroduce a FindQt5.cmake like that, this is bound to have weird interactions.